content/4.validators/1.running-a-node/5.join-a-network/4.sync-from-snapshot.md

```diff
@@ -14,7 +14,10 @@ In order to sync your node through a snapshot, you will need to:
 - Start your node.
 
 ## Download the snapshot
-Currently there are few snapshot maintained by the community that can be used, provided by the following teams:
+You can download the latest snapshot from:
+<a href="https://snapshots.archway.tech" target="_blank">snapshots.archway.tech</a>.
+
+Also, you can use the snapshot maintained by the community:
```

Comment:
I still think we should put a note saying we have no control over the community snapshots so use at your own risk.

Reply:
thanks for the feedback, i have added a disclaimer